modules/cert-manager-acme-dns01-ambient-aws.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-acme-dns01-ambient-aws_{context}"]
+= Configuring an ACME issuer to solve DNS-01 challenges by using ambient credentials on AWS
+
+You can use {cert-manager-operator} to set up an ACME issuer to solve DNS-01 challenges by using ambient credentials on AWS. This procedure uses _Let's Encrypt_ as the ACME CA server and shows how to solve DNS-01 challenges with Amazon Route 53.
+
+.Prerequisites
+
+* If you are using the AWS Security Token Service cluster, you followed the instructions from the _Configuring cloud credentials for the cert-manager Operator for Red Hat OpenShift for the AWS Security Token Service cluster_ section.
+* If you are not using the AWS Security Token Service cluster, you followed the instructions from the _Configuring cloud credentials for the cert-manager Operator for Red Hat OpenShift on AWS_ section.
+
+.Procedure
+
+. Optional: Override the nameserver settings for the DNS-01 self check:
```

Comment:
No `:` since there is a line following between this instruction and the list of substeps. Otherwise you would use it :slightly_smiling_face: 
```suggestion
. Optional: Override the nameserver settings for the DNS-01 self check.
```